src/server/endpoints.js

```diff
@@ -41,7 +41,7 @@ export const createGame = (gameServer) => async (ctx) => {
     const gameId = r.body.matchID;
     const credentials = [];
 
-    for (var i = 0; i < ctx.request.body.players; i++) {
+    for (let i = 0; i < ctx.request.body.players; i++) {
```

Comment:
As far as I see it, `endpoints.js` is the only file in the `server` code that did not get migrated to TypeScript. Is there a reason for this?

Reply:
agreed.